package.json

```diff
@@ -101,14 +101,13 @@
       "<rootDir>/src.*",
       "<rootDir>/node_modules.*",
       "<rootDir>/app/src.*",
-      "<rootDir>/app/dist.*",
       "<rootDir>/app/node_modules.*"
     ],
     "watchPathIgnorePatterns": [
       "<rootDir>/src.*",
       "<rootDir>/tsconfig.json",
+      "<rootDir>/app/node_modules.*",
```

Comment:
Did you encounter "too many watch handles" errors on your machine?

- If no, let's not ignore if it's not necessary for the watcher to function. Might be nice when battling a dependency bug to have a working live reload on deps as you fiddle with the dependency bug
- If yes, please exclude `"<rootDir>/node_modules.*",` too